janus_core/calculations/md.py

```diff
@@ -1106,6 +1112,13 @@ def _run_dynamics(self) -> None:
         if self.ramp_temp:
             heating_steps = int(self.temp_time // self.timestep)
 
+            if self.logger and not np.isclose(self.temp_time % self.timestep, 0.0):
```

Comment:
Just to note, this can give slightly strange results e.g. `100 % 0.1` - > `0.09999999999999445`, which so this returns `False` with the default tolerances.

It's not necessarily a problem, since the log info doesn't hurt, and for most sensible time steps, this doesn't seem to be an issue.